Bundle/ConfigBundle/Resources/translations/validators.en.xliff

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file source-language="en" target-language="en" datatype="plaintext" original="not.available">
+    <body>
+      <trans-unit id="5644329337838199438899" resname="victoire.config.global.metaTitlePattern.invalid">
+          <source>victoire.config.global.metaTitlePattern.invalid</source>
+          <target><![CDATA[You need at least %page.title%]]></target>
```

Comment:
What error message will be displayed to the user? It looks like something is missing.